common/production_methods/11_private_infrastructure.txt

```diff
@@ -0,0 +1,298 @@
+﻿pm_steam_trains = {
+	texture = "gfx/interface/icons/production_method_icons/trains_steam.dds"
+	pollution_generation = 15
+
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_engines_add = 10
+			building_input_coal_add = 10
+			building_output_transportation_add = 50
+		}
+		level_scaled = {
+			building_employment_laborers_add = 3000
+			building_employment_machinists_add = 1000
+			building_employment_clerks_add = 750
+		}
+	}
+
+	state_modifiers = {
+		workforce_scaled = {
+			state_infrastructure_add = 20
+		}
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			market_land_trade_capacity_add = 1
```

Comment:
these land_trade_capacity need to be divide by 10. At least according to my sense of balance.

Reply:
Why do they seem too high right now? theoretically most nations will only ever build about 400-600 railways max (assuming like Britain) by the 1920s, and will be trading in the thousands (sometimes tens of thousands) by that point. I think paradox's nerf to land trade capacity is especially painful for the nations that are landlocked like Switzerland, who are otherwise incapable of trading because of a lack of convoys.

Reply:
Land trade capacity is not just convoys, it gives a flat capacity to every single trade route at the same time. So when converting to convoys, you want to at least multiply it by the amount of goods you can trade. In addition you want to multiply by the amount of trading partners a country can have. Furthermore trade capacity is the equivalent of a raw 2 convoys.

So 1000 land_trade_capacity will give 2000 convoys for every single trade route.

Reply:
Understood, i'll reduce the amounts then and make the decimal visible